lua/telescope/_extensions/cheat.lua

```diff
@@ -0,0 +1,78 @@
+local has_telescope, telescope = pcall(require, 'telescope')
+if not has_telescope then
+  error('This plugins requires nvim-telescope/telescope.nvim')
+end
+
+local curl = require("plenary.curl")
+local finders = require("telescope.finders")
+local make_entry = require("telescope.make_entry")
+local pickers = require("telescope.pickers")
+local sorters = require("telescope.sorters")
+local conf = require("telescope.config").values
+local previewers = require("telescope.previewers")
+local utils = require('telescope.utils')
+local putils = require('telescope.previewers.utils')
+local actions = require('telescope.actions')
+local defaulter = utils.make_default_callable
+
+local query = function(qry)
+  local query, filetype, results
+  if qry == nil or vim.tbl_isempty(qry) then
+    query = ":list"
+  elseif qry.child == nil then
+    query = string.format("%s/:list", qry.ft)
+    filetype = qry.ft
+  else
+    query = string.format("%s/%s", qry.ft, qry.child)
+    filetype = qry.ft
+  end
+  res = curl.get("cht.sh/"..query.."?T") .body or ""
+
+  return {
+    res = vim.split(res, "\n"),
+    ft = filetype or ""
+  }
+end
+
+local cheat_fd = function(opts) -- TODO Make it non blocking
+  opts = opts or {}
+  local res = query({ft = opts.ft, child = opts.child})
+  pickers.new(opts, {
+    prompt_title = 'Cheats',
+    finder = finders.new_table{ results = res.res },
+    sorter = conf.generic_sorter(opts),
+    ft = res.ft, -- incase we want to have recusive finders.
+    -- previewer = previewers.new_buffer_previewer{
+    --   get_buffer_by_name = function(_, entry)
+    --     return entry.value
+    --   end,
+    --   define_preview = function(self, entry, status) -- FIXME:
+    --     putils.with_preview_window(status, nil, function()
+    --       local output = query{ft = self.ft, child = entry.value}
+    --       vim.api.nvim_buf_set_lines(self.state.bufnr, 0, -1, true, output)
```

Comment:
shouldn't this be `output.res` instead of `output` ? Because `output` is a table, not a string.
I changed it to `output.res` and it works.

Reply:
Oh wow it did, nice catch pushing new version ...